### DISCUSSION TITLE
Nodejs

### DISCUSSION THREAD

**Discussion by mimiMonads** [mimiMonads/vixeny#35] [Ideas]
Simple question, should we give support to it?

pros:
- User base
- Stable
- Nodejs is giving support to ts
- Relatively easy to implement
- It just has to work, it's not necessary to get it as optimized as deno / bun

cons:
- We have to add a serve for the Request -> Response cycle
- `Create` also has to support it
- Testing for nodejs

**Comment by aquapi** [comment]
Supporting node is kind of going against the performance and memory advantage of Vixeny.
